tests/Feature/ReturnRegisterDataTest.php

```diff
@@ -81,7 +81,24 @@ public function updatePasswordTest()
             'token' => $reset->token,
             'password' => $password,
             ];
-        $response = $this->post('/api/auth/update', $data);
+        $response = $this->post('/api/update', $data);
         $response->assertStatus(200);
     }
+    /** @test */
+    public function updateTest()
+    {
+        $user = User::factory()->create([
+            'id' => 40,
+            'name' => 'Qwertyqweqwe',
+            'email' => 'qwerty2@gmail.com',
+        ]);
+        $data = [
+            'name' => 'Qrreewert',
+            'email' => 'qwerty112@gmail.com',
+        ];
+        $this->actingAs($user, 'api');
+        $response = $this->put('/api/auth/users/40', $data);
+        $response->assertStatus(201);
+        $response->assertJsonStructure(['token']);
```

Comment:
where did you check that values were updated? 
p.s. you can use different assert methods, for example, $this->assertEquals or NotEquals and so on.